src/support/AdminContext.js

```diff
@@ -88,16 +90,16 @@ export class AdminContext {
         }
         const { code: { owner, repo } } = config;
         info.withCode(owner, repo);
-        this.attributes.config = config;
+        attributes.config = config;
       }
     }
-    if (this.attributes.orgConfig === undefined) {
+    if (attributes.orgConfig === undefined) {
       const { org } = info;
       if (org) {
-        this.attributes.orgConfig = await loadOrgConfig(this, org);
+        attributes.orgConfig = await loadOrgConfig(this, org);
       }
     }
-    return this.attributes.orgConfig;
+    return attributes.config;
```

Comment:
org config is optional

Reply:
yes, but you only need to load it, if the request doesn't address a site